com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -70,6 +70,29 @@ private enum SMAAStage
         readonly GraphicsFormat m_LutFormat;
         readonly HableCurve m_HableCurve;
 
+        //Viewport information
+
+        Vector2Int m_FullRes = new Vector2Int(1,1);
+        Vector2Int m_DownsampledRes = new Vector2Int(1,1);
+
+        internal enum ResolutionGroup
+        {
+            Downsampled,
+            Full
+        }
+
+        internal ResolutionGroup ResGroup { set; get; }
+
+        internal Vector2Int Viewport { get { return  ResGroup == ResolutionGroup.Full ? m_FullRes : m_DownsampledRes; } }
+
+        internal enum UpsamplerScheduleType
+        {
+            BeforePost,
+            AfterPost,
+        }
+
+        internal UpsamplerScheduleType UpsamplerSchedule { set; get; }
```

Comment:
I think this information needs to stay with the dynamic res handler

Reply:
So the schedule type goes back to the DRS handler? Itd have to become public then

Reply:
Yeah I think that makes sense

Reply:
moved to drs handler